onebusaway-android/src/main/java/org/onebusaway/android/io/elements/ObaRegion.java

```diff
@@ -35,6 +35,15 @@
         public double getLonSpan();
     }
 
+    public interface Open311Servers {
```

Comment:
If this interface represents a single Open311 server, it should be `Open311Server`.